Userland/Libraries/LibGfx/ImageFormats/CCITTDecoder.h

```diff
@@ -23,4 +23,26 @@ namespace Gfx::CCITT {
 // Section 10: Modified Huffman Compression
 ErrorOr<ByteBuffer> decode_ccitt_rle(ReadonlyBytes bytes, u32 image_width, u32 image_height);
 
+// While this is named for a CCITT context, this struct holds data like TIFF's T4Options tag
+struct Group3Options {
```

Comment:
nit: commit message reads weird. Either add a comma like so: "These two fields, T4Options and T6Options, are described in:" or do "The two fields T4Options and T6Options are described in:".

Reply:
The main issue was that I didn't update the commit message after changing the commit to only include T4Options.